website/docs/blog.mdx

```diff
@@ -227,6 +227,7 @@ author_title: Co-creator of Docusaurus 1
 author_url: https://github.com/JoelMarcey
 author_image_url: https://github.com/JoelMarcey.png
 ---
+
```

Comment:
@Josh-Cena any idea why `yarn prettier` add back those spaces?

I may did something wrong when merging, maybe some config you added was deleted?

Reply:
oh nevermind it's just I didn't run the `prettier-docs` command.

For weird reasons my IDE still reformats the old way on save 😓